stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/ChannelListDiffCallback.kt

```diff
@@ -0,0 +1,100 @@
+package com.getstream.sdk.chat.adapter
+
+import androidx.recyclerview.widget.DiffUtil
+import com.getstream.sdk.chat.utils.extensions.computeLastMessage
+import com.getstream.sdk.chat.utils.extensions.getOtherUsers
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.ChannelUserRead
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.livedata.ChatDomain.Companion.instance
+
+internal class ChannelListDiffCallback @JvmOverloads constructor(
+    private val oldList: List<Channel>,
+    private val newList: List<Channel>,
+    private val currentUser: User = instance().currentUser
```

Comment:
I would rather call `ChatDomain.instance().currentUser` instead of importing instance. I think it will be more readable

Reply:
Done